src/promptflow/promptflow/_sdk/_start_trace.py

```diff
@@ -0,0 +1,61 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import os
+import uuid
+
+from opentelemetry import trace
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+from promptflow._sdk._constants import TRACE_SESSION_ID_ENV_VAR
+from promptflow._sdk._service.utils.utils import get_port_from_config
+from promptflow._utils.logger_utils import get_cli_sdk_logger
+
+_logger = get_cli_sdk_logger()
+
+
+def start_trace():
```

Comment:
we might need to put this file out from promptflow._sdk as @trace need to reference to this.

Reply:
move out, and place it under `/_traces` for now.